InstantSearch.js/query-suggestions/src/app.js

```diff
@@ -1,125 +1,191 @@
-/* global $ instantsearch algoliasearch */
+import { autocomplete } from '@algolia/autocomplete-js';
+import algoliasearch from 'algoliasearch/lite';
+import instantsearch from 'instantsearch.js';
+import { highlight } from 'instantsearch.js/es/helpers';
+import {
+  connectAutocomplete,
+  connectSearchBox,
+  connectRefinementList,
+} from 'instantsearch.js/es/connectors';
+import {
+  configure,
+  index,
+  hits,
+  refinementList,
```

Comment:
```suggestion
```

Reply:
(Looks like only a custom refinementList is used.)

Reply:
Indeed—addressed in 55acdf4c5b045d3fde0ca35d908272de0909177d.